src/chatClient.cpp

```diff
@@ -393,6 +393,42 @@ int Client::importMessages(const char *externalDbPath)
         chatd::Chat &chat = chatroom->chat();
         karere::Id chatid = chatroom->chatid();
 
+        // Import last seen from external db
+        Id lastSeenId;
+        SqliteStmt stmtLastSeen(dbExternal, "select last_seen from chats where chatid=?");
+        stmtLastSeen << chatid;
+        stmtLastSeen.stepMustHaveData();
```

Comment:
What if the chatroom has no history?? In that case, I guess there's no  `SEEN` for the given `chatid`